README.md

```diff
@@ -13,33 +13,23 @@ remote Opam repository:
 
 ## Layout of This Repository
 
-Read this to understand how to make changes
-
-    +-- compilers                       part of Opam repo (empty)
-    +-- LICENSE
-    +-- packages                        part of Opam repo
-    |   +-- upstream                    all upstream packages
-    |   |   +-- angstrom.0.1.1
-    |   |   |   +-- descr
-    |   |   |   +-- findlib
-    |   |   |   +-- opam
-    |   |   |   \-- url
-    |   |   \-- zed.1.4
-    |   |       +-- descr
-    |   |       +-- findlib
-    |   |       +-- opam
-    |   |       \-- url
-    |   \-- xs                          all XenServer packages
-    |       +-- message-switch.jonludlam#ppx
-    |       |   +-- descr
-    |       |   +-- opam
-    |       |   \-- url
-    |       \-- xenstore_transport.0.9.4
-    |           +-- descr
-    |           +-- findlib
-    |           +-- opam
-    |           \-- url
-    +-- README.md
+Packages are organised into namespaces for organisational aspects. They
```

Comment:
Would it be better to move this paragraph with the bullet points to the bottom, below the layout and general description of `packages/`? Maybe that would improve the flow of the text.